src/main/java/org/reaktivity/nukleus/amqp/internal/stream/AmqpServerFactory.java

```diff
@@ -1185,20 +1208,20 @@ private void putAnnotation(
             case 1:
                 AmqpULongFW id = amqpULongRW.wrap(valueBuffer, valueOffset.value, valueBuffer.capacity())
                     .set(item.key().id()).build();
-                valueOffset.value += id.limit();
+                valueOffset.value += id.sizeof();
                 AmqpByteFW value1 = amqpByteRW.wrap(valueBuffer, valueOffset.value, valueBuffer.capacity())
                     .set(item.value().bytes().value().getByte(0)).build();
-                valueOffset.value += value1.limit();
-                amqpMapRW.entry(k -> k.setAsAmqpULong(id), v -> v.setAsAmqpByte(value1));
+                valueOffset.value += value1.sizeof();
+                annotationRW.entry(k -> k.setAsAmqpULong(id), v -> v.setAsAmqpByte(value1));
                 break;
             case 2:
                 AmqpSymbolFW name = amqpSymbolRW.wrap(valueBuffer, valueOffset.value, valueBuffer.capacity())
                     .set(item.key().name()).build();
-                valueOffset.value += name.limit();
+                valueOffset.value += name.sizeof();
                 AmqpByteFW value2 = amqpByteRW.wrap(valueBuffer, valueOffset.value, valueBuffer.capacity())
                     .set(item.value().bytes().value().getByte(0)).build();
-                valueOffset.value += value2.limit();
-                amqpMapRW.entry(k -> k.setAsAmqpSymbol(name), v -> v.setAsAmqpByte(value2));
+                valueOffset.value += value2.sizeof();
+                annotationRW.entry(k -> k.setAsAmqpSymbol(name), v -> v.setAsAmqpByte(value2));
```

Comment:
Let's use the corresponding generated `KIND` constants in the case statements instead of `1` and `2`.